### PR TITLE
Update ServiceInvocationTest to use local rules instead of contacting BLZ Service

### PR DESCRIPTION
**Pull request by t-burch**

### CODE REVIEW

**Review by predic8 — DISMISSED** [review]

File: core/src/test/java/com/predic8/membrane/core/IntegrationTestsWithInternet.java [hunk 1]
```diff
@@ -32,7 +31,6 @@
         MethodTest.class,
         RegExReplaceInterceptorTest.class,
         LoadBalancingInterceptorTest.class,
```
Comment (predic8) [inline core/src/test/java/com/predic8/membrane/core/IntegrationTestsWithInternet.java:33]: Do you have to include the SeviceIncovationTest class?

Comment (t-burch) [reply core/src/test/java/com/predic8/membrane/core/IntegrationTestsWithInternet.java:33]: This was a duplicate test doing the same thing as the other I removed it

**Comment by t-burch** [comment]
Actually the InterceptorInvocationTest seems to be very similar but maybe tests something different?

**Comment by rrayst** [comment]
@t-burch 11 test failures...

**Review by rrayst — APPROVED** [review]

**Pull request merged**